src/test/kotlin/uk/gov/dwp/dataworks/services/TaskDeploymentServiceTest.kt

```diff
@@ -0,0 +1,65 @@
+package uk.gov.dwp.dataworks.services
+
+import org.junit.Assert
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.boot.test.context.SpringBootTest
+import org.springframework.test.context.ContextConfiguration
+import org.springframework.test.context.junit4.SpringRunner
+import software.amazon.awssdk.services.elasticloadbalancingv2.model.DescribeRulesResponse
+import software.amazon.awssdk.services.elasticloadbalancingv2.model.Rule
+import uk.gov.dwp.dataworks.Application
+import java.lang.Exception
+import java.util.*
+
+
+@RunWith(SpringRunner::class)
+@ContextConfiguration(classes = [Application::class])
+@SpringBootTest(properties = ["orchestrationService.cognito_user_pool_id=id"])
+class TaskDeploymentServiceTest{
+
+    @Autowired
+    private lateinit var configService: ConfigurationService
+    @Autowired
+    private lateinit var authService: AuthenticationService
+    @Autowired
+    private lateinit var taskDeploymentService: TaskDeploymentService
+
+
+    val nonConsecutiveCol : Collection<Rule> = listOf(Rule.builder().priority("1").build(), Rule.builder().priority("2").build(), Rule.builder().priority("4").build())
+
+    @Test
+    fun `testPriorityNumberForNoRuleSetExpectOne`(){
```

Comment:
The use of ` is such that spaces can be put into a method name. We should either add spaces to the method names or remove the backticks.